frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketsTopPanel/statusProperty/statusProperty.component.tsx

```diff
@@ -38,7 +38,9 @@ export const StatusProperty = ({ onBlur, readOnly }: StatusPropertyProps) => {
 	const ticketView = TicketsCardHooksSelectors.selectView();
 	const templateId = ticketView === TicketsCardViews.New ? templateIdNewTicket : templateIdExistingTicket ?? templateIdTabularView;
 
-	const statusConfig = TicketsHooksSelectors.selectStatusConfigByTemplateId(containerOrFederation, templateId);
+	const statusConfig = templateIdTabularView
+		? ProjectsHooksSelectors.selectStatusConfigByTemplateId(templateId)
+		: TicketsHooksSelectors.selectStatusConfigByTemplateId(containerOrFederation, templateId);
```

Comment:
can we change TicketsHooksSelectors.selectStatusConfigByTemplateId so it also takes information from the projecttemplates?